### PR TITLE
Introduce Compose Multiplatform to iOS with ContributorScreen

### PR DESCRIPTION
**Pull request by takahirom**
## Issue
- close https://github.com/DroidKaigi/conference-app-2023/issues/991

## Overview (Required)

- I want to fix the issue where I can't display each contributor in SafariView.

https://github.com/DroidKaigi/conference-app-2023/assets/1386930/dd53954a-e066-4bdb-8356-104a64af96d8

## Links
- 

## Screenshot (Optional if screenshot test is present or unrelated to UI)
Before | After
:--: | :--:
 | 

## Movie (Optional)
Before | After
:--: | :--:
<video src="" width="300" > | <video src="" width="300" >

### CODE REVIEW

**Comment by takahirom** [comment]
Thanks for your fix @tkhs0604 ! We managed to use the Compose Multiplatform!

File: app-ios/Modules/Sources/Contributor/ContributorView.swift [hunk 1]
```diff
@@ -1,18 +1,25 @@
...
+        Group {
+//            ContributorSwiftUIView { url in
+//                presentingURL = .init(string: url)
+//            }
+
+            ContributorComposeView { url in
+                presentingURL = .init(string: url)
+            }
+        }
         ContributorComposeView(
```
Comment (tkhs0604) [inline app-ios/Modules/Sources/Contributor/ContributorView.swift:22]: @ry-itto 
Is it unnecessary now? 👀

Comment (tkhs0604) [reply app-ios/Modules/Sources/Contributor/ContributorView.swift]: Removed https://github.com/DroidKaigi/conference-app-2023/pull/1054/commits/ec4894414ccce15c7bb3c465bd7903a743e9438c

**Comment by tkhs0604** [comment]
📝 The color of bottom navigation was improved by bumping some compose related plugin/library versions https://github.com/DroidKaigi/conference-app-2023/pull/1054/commits/fae3ab094e69ddb11174f04ce360d22da6af950e
image

**Comment by tkhs0604** [comment]
📝 Seems like we need to apply the app's theme explicitly by adding KaigiTheme at the root of ContributorsScreen.

**Review by tkhs0604 — APPROVED** [review]
An issue for the remaining task was created, so we can now merge it 👀

**Pull request merged**